packages/taquito-tzip16/src/viewKind/michelson-storage-view.ts

```diff
@@ -169,7 +170,26 @@ export class MichelsonStorageView implements View {
             balance: '0'
         };
 
-        const result: any = await this.rpc.runCode(viewScript as any);
+        let result: any;
+        try {
+          result = await this.rpc.runCode(viewScript as any);
+        } catch(error: any) {
+          const parsedError = JSON.parse(error.body);
+
+          parsedError.forEach((x: any) => {
+            if (x.id.includes('script_rejected')) {
+              throw new ViewSimulationError(`Failed to execute the contract view ${parsedError}`)
+            } 
+          });
+          throw new HttpResponseError(
```

Comment:
consider replacing by `throw error`

Reply:
I'm guessing we want that for the sake of generality?